File: Chatbot WCF IIS HOST/wcf 2/Chatbot/Models/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Runtime.Serialization;

namespace Chatbot.Models
{
    [DataContract]
    public class Task : Rule
    {
        [DataMember]
        public string method { get; set; }
        [DataMember]
        public string api { get; set; }
        [DataMember]
        public List<SequenceMessage> sequenceMessages { get; set; } = new List<SequenceMessage>();
        [DataMember]
        public string responseType { get; set; }
        [DataMember]
        public string graphType { get; set; }

    }
}

File: Chatbot Angular/Chatbot Desktop Version/Chat/src/app/classes/Task.ts
import { Rule } from "./Rule";
import { SequenceMessage } from "./SequenceMessage";
import { Trigger } from "./Trigger";

export class Task extends Rule{
    constructor(
        public id?,
        public title?: string,
        public description?: string,
        public ruleType?:string,
        public triggers?:Trigger[],
        public sequenceMessages?:SequenceMessage[],
        public method?:string,
        public api?:string,
        public responseType?:string,
        public graphType?:string){
            
            super(id,title,description,ruleType,sequenceMessages);

    }
    
}